specification/personal-demographics.yaml

```diff
@@ -779,7 +786,7 @@ paths:
         ```json
         {
             "patches": [
-              { "op": "add", "path": "/deceasedDateTime", "value": "2020-01-01" }
+              { "op": "add", "path": "/deceasedDateTime", "value": "2025-01-01" }
```

Comment:
Minor nitpick, deceased date can't be in the future, so this one is fine being in the past

Reply:
Ah good point! I'll get that fixed now